### ISSUE TITLE
Incorrect assumption that non-Marshallable Comparables are scalars

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Wire#751]
At `Wires.java:1120`
```
            if (Comparable.class.isAssignableFrom(aClass))
                return ANY_SCALAR;
```

In practice, two POJOs may be Comparable.

**Comment by alamar** [comment]
```
public class Issue751Test extends WireTestCommon {

    public static class One extends SelfDescribingMarshallable {
        Comparable<?> text;

        public One(Comparable<?> text) {
            this.text = text;
        }
    }

    public static class Two implements Comparable<Two> {
        Comparable<?> text;

        public Two(Comparable<?> text) {
            this.text = text;
        }

        @Override
        public int compareTo(@NotNull Issue751Test.Two o) {
            return text.hashCode() - o.text.hashCode();
        }
    }

    public static class Three extends SelfDescribingMarshallable {
        private One one;
        private Two two;

        public Three(One one, Two two) {
            this.one = one;
            this.two = two;
        }
    }

    @Test
    public void comparableField() {
        Wire wire = new YamlWire(Bytes.allocateElasticOnHeap());
        wire.write("first").object(new Three(
                new One("hello"), new Two(42)));

        System.err.println(wire.read("first").object());
    }
}
```
reproducer

**Comment by alamar** [comment]
Not a problem in practice since class should implement `Marshallable`/`Serializable` as well.
Correctly handling raw classes is a goodwill of Wire, and no behavior change is likely possible or needed.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.25ea2](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.25ea2), [BOM-2.25ea10](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.25ea10)
